syntaxes/kotlin.tmLanguage.json

```diff
@@ -19,10 +19,13 @@
     ],
     "repository": {
         "import": {
-            "begin": "\\b(import)\\b\\s*",
+            "begin": "\\b(import)\\b\\s*([\\w+.]*\\w+)?\\s*",
```

Comment:
Hm, I am not entirely convinced this rule is correct. This will match the package too in imports and highlight it as if it were a modifier:

<img width="550" alt="image" src="https://github.com/fwcd/vscode-kotlin/assets/30873659/8b94e8a4-1453-4d11-bee9-348316e77171">

For comparison, this is the current state which, IMO, is more correct:

<img width="606" alt="image" src="https://github.com/fwcd/vscode-kotlin/assets/30873659/a8b0d8cc-e556-42b7-97bb-a7708821bb30">

Reply:
I cannot reproduce this. I have tried with both my fork of the extension and by creating an empty extension with only the grammar from the PR:
![image](https://github.com/fwcd/vscode-kotlin/assets/25014042/313aac31-37ed-4a5a-be98-947810882449)

Reply:
Yes, your color scheme is different, but the package shouldn't have scope `storage.modifier.*`

Reply:
I have looked on how the java grammar handles this. It looks a bit more sophisticated than this. I am nut sure if I will be able to fix it this week